index.ts

```diff
@@ -154,18 +155,18 @@ export function playTrackInContext(player: PlayerName, params: any[]) {
     return musicCtr.playTrackInContext(player, params);
 }
 
-/**
+/** review name
  * Initiate and play the specified Spotify device
- * @param device_id {string}
- * @param play {boolean}
+ * @param device_id {string }
+ * @param play {boolean }
  */
 export function playSpotifyDevice(device_id: string, play: boolean = true) {
```

Comment:
This is probably ok, but confuses me a little b/c the name and description seem to not exactly match the intent.
If you pass `false` for the play param what would happen? In other words, if the function is described as initiate and play, then maybe it should not even take the second param, just play.